libs/ui/src/lib/notification/Toast/Toast.container.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-cycle
+import Toast from './Toast';
+
+import { ToastContainerProps } from './Toast.props';
+
+import * as S from './Toast.style';
+
+const ToastContainer = ({ messages }: ToastContainerProps) => {
+  return (
+    <S.ToastContainer>
+      {messages.map(message => (
+        <Toast key={message.id} message={message} />
```

Comment:
aqui você poderia para o Toast todas as suas props de forma destruturada:
```suggestion
        <Toast key={message.id} {...message} />
```